packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -95,9 +95,17 @@ function useLoadStory({ storyId, shouldLoad, restore, isDemo }) {
         const publisherLogo = {
           id: embedded?.['wp:publisherlogo']?.[0].id || 0,
```

Comment:
Maybe extract `embedded?.['wp:publisherlogo']?.[0]` into a separate constant for readability?

Reply:
I was thinking about that but we don't do that generally(?)
![image](https://user-images.githubusercontent.com/21036224/133097811-f93fed3e-4d00-4936-8c90-8c30aac9468d.png)

Reply:
Hm, okay. Not sure what's the reason 🤔